raiden/connection_manager.py

```diff
@@ -126,34 +143,48 @@ def leave(self, wait_for_settle=True, timeout=30):
                 c.partner_address) for c in open_channels]
             for channel in channel_specs:
                 try:
-                    self.api.close(*channel),
-                except RuntimeError:
+                    self.api.close(*channel)
+                except:
```

Comment:
I would say it's not, to me it breaks the [Principle of least astonishment](https://en.wikipedia.org/wiki/Principle_of_least_astonishment) (after all we both thought the same thing, didn't we?)